### PR TITLE
remove showing help twice for incorrect commands

### PR DESCRIPTION
**Pull request by girishramnani**
**What type of PR is this?**
> Uncomment only one ` /kind` line, and delete the rest.
> For example, `> /kind bug` would simply become: `/kind bug`

> /kind api-change

/kind bug

> /kind cleanup
> /kind deprecation
> /kind design
> /kind failing-test
> /kind feature
> /kind flake
> /kind code-refactoring
>
> Documentation changes: Please include [skip ci] in your commit message as well
> /kind documentation
> [skip ci]

**What does does this PR do / why we need it**:
see $SUBJECT

**Which issue(s) this PR fixes**:

Fixes https://github.com/openshift/odo/issues/3750

**PR acceptance criteria**:

- [ ] Unit test 

- [ ] Integration test 

- [ ] Documentation 

- [x] I have read the [test guidelines](https://github.com/openshift/odo/blob/master/docs/dev/test-architecture.adoc)

**How to test changes / Special notes to the reviewer**:
```
odo hello 
```
should only print help once

### CODE REVIEW

File: pkg/odo/cli/cli.go [hunk 1]
```diff
@@ -272,6 +272,5 @@ func ShowHelp(cmd *cobra.Command, args []string) error {
 		return nil
 	}
 
-	_ = cmd.Help()
```
Comment (mik-dass) [inline pkg/odo/cli/cli.go:275]: I think we can have a integration test where we check for only one occurence of `Invalid command - see available commands/subcommands above`

**Comment by amitkrout** [comment]
@girishramnani Can you please attach the screenshot or text output of before and after change. I am asking because it's a UX change, so output before and after would really be helpful.

File: pkg/odo/cli/cli.go (hunk 1 above)
Comment (amitkrout) [reply pkg/odo/cli/cli.go:275]: Yes, may be it will be helpful to avoid future regression.

Comment (girishramnani) [reply pkg/odo/cli/cli.go:275]: added the test - I couldn't use `Invalid command - see available commands/subcommands above` as that shows up twice already and hence went for a unique string

**Review by mik-dass — APPROVED** [review]
/approve

**Pull request merged**